readthedocs/settings/base.py

```diff
@@ -784,6 +784,10 @@ def DOCKER_LIMITS(self):
     DJSTRIPE_FOREIGN_KEY_TO_FIELD = "id"
     DJSTRIPE_USE_NATIVE_JSONFIELD = True  # We recommend setting to True for new installations
 
+    # We are managing the subscriber relationship by ourselves,
+    # since we have subscriptions attached to an organization or gold user.
+    DJSTRIPE_SUBSCRIBER_CUSTOMER_KEY = None
```

Comment:
Is there more information here? I don't quite understand this comment -- is it disabling some feature of djstripe? Are we going to turn this back on in the future?

Reply:
It's disabling the feature, yes (well, partially, since stripe still creates a relationship to the user object), and we won't be using this in the future. I'll expand the comment.